packages/mobile/src/invite/JoinCelo.tsx

```diff
@@ -136,6 +139,7 @@ export class JoinCelo extends React.Component<Props, State> {
       return
     }
 
+    this.props.setPromptZeroSync(true) // Allow zero sync prompt after Welcome screen
```

Comment:
Why set it manually instead of making true the default value?

Reply:
We don't want to prompt the switch until after the user has gone through the Welcome screen. By having it default false we prevent the pop up from showing up at the beginning of the new user UX